docs/architecture/self-hosted/installation-helm/install-on-aws.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Installation on AWS
+id: install-on-aws
+description: Use this guide to help when installing Prophecy on AWS.
+tags:
+  - deployment
+  - installation
+  - aws
+---
+
+import TOCInline from "@theme/TOCInline";
+
+If you are deploying Prophecy with the private SaaS option, you need to install Prophecy in your Virtual Private Cloud (VPC). Use this guide to help when installing Prophecy on AWS.
+
+<TOCInline toc={toc} maxHeadingLevel={2} />
+
+## Get started
+
+In this guide, we'll walk you through how to install Prophecy on an [EKS cluster](https://aws.amazon.com/eks/) in AWS. Note that this is just one way to perform the installation.
```

Comment:
Instead of phrasing it as "just one way" should we say "the recommended way"?

Reply:
well, i don't think that this *is* the recommended way. apparently azure and aws are somewhat here. maybe i can say "one recommended way"